docs/user/visualize.asciidoc

```diff
@@ -7,74 +7,79 @@ _Visualize_ enables you to create visualizations of the data from your {es} indi
 
 {kib} visualizations are based on {es} queries. By using a series of {es} {ref}/search-aggregations.html[aggregations] to extract and process your data, you can create charts that show you the trends, spikes, and dips you need to know about.
 
+To begin, open *Visualize*, then click *Create new visualization*.
+
 [float]
-[[create-a-visualization]]
-== Create visualizations
+[[visualization-types]]
+== Types of visualizations
+
+To help you choose the best visualization for your data, {kib} supports many types of visualizations.
 
-. Open *Visualize*.
-. Click *Create new visualization*.
-. Choose the visualization type:
-+
-* Basic charts
-[horizontal]
 <<lens,Lens>>::
 Quickly build several types of basic visualizations by simply dragging and dropping the data fields you want to display.
 
-* *<<most-frequent,Most frequently used visualizations>>*
-[horizontal]
-Line&comma; area, and bar charts:: Compare different series in X/Y charts.
+<<most-frequent,Most frequently used visualizations>>::
 
-Pie chart:: Display each source contribution to a total.
+* *Line, area, and bar charts* &mdash; Compare different series in X/Y charts.
 
-Data table:: Flattens aggregations into table format.
+* *Pie chart* &mdash; Display each source contribution to a total.
 
-Metric:: Display a single number.
+* *Data table* &mdash; Flattens aggregations into table format.
 
-Goal and gauge:: Display a number with progress indicators.
+* *Metric* &mdash; Display a single number.
 
-Tag cloud:: Display words in a cloud, where the size of the word corresponds to its importance.
+* *Goal and gauge* &mdash; Display a number with progress indicators.
+
+* *Tag cloud* &mdash; Display words in a cloud, where the size of the word corresponds to its importance.
 
-* *Time series optimized*
-[horizontal]
 <<TSVB,TSVB>>:: Visualize time series data using pipeline aggregations.
 
 <<timelion,Timelion>>:: Compute and combine data from multiple time series
 data sets.
 
-* *Maps*
-[horizontal]
-<<maps,Elastic Maps>>:: The most powerful way of visualizing map data in {kib}.
+<<visualize-maps, Maps>>::
+* *<<maps,Elastic Maps>>* &mdash; The most powerful way of visualizing map data in {kib}.
```

Comment:
map data > geospatial data

Maybe make this bullet point parallel with the rest